src/data/links.yaml

```diff
@@ -79,6 +80,7 @@
     - carbon-reduction
   tags: ['carbon-reduction']
   description: Emails contribute to 986,000 tonnes of CO2 being dumped into the atmosphere every day, that's 4g per email. Unsubscribe from unwanted subscription emails and reduce your carbon footprint.
+  image: http://placekitten.com/300/200
```

Comment:
Hey @rapzo - can you change these to `https`? They're causing the build to break currently.  I know these images will all be replaced eventually, but this will be a big help to get us there.  Thank you!

Reply:
On it. But i'm not very comfortable with my solution... Are the pictures gonna be downloaded previously, like the other ones in the `src/images` folder? Because that would change the the query.

Reply:
Done.